servers/quarkus-server/src/main/java/org/projectnessie/server/authn/NessieHttpAuthenticator.java

```diff
@@ -47,17 +52,70 @@
 @ApplicationScoped
 public class NessieHttpAuthenticator extends HttpAuthenticator {
 
-  @Inject IdentityProviderManager identityProvider;
-
+  private final IdentityProviderManager identityProvider;
   private final boolean authEnabled;
 
+  // Demanded by Quarkus/Arc
+  public NessieHttpAuthenticator() {
+    super(
+        null,
+        null,
+        new Instance<>() {
```

Comment:
nit: Could you move this to a named class... it seems too verbose for an in-line impl. :shrug: